ArduCopter/esc_calibration.cpp

```diff
@@ -153,7 +153,7 @@ void Copter::esc_calibration_setup()
     // clear esc flag for next time
     g.esc_calibrate.set_and_save(ESCCAL_NONE);
 
-    if (motors->get_pwm_type() >= AP_Motors::PWM_TYPE_ONESHOT) {
+    if (motors->is_normal_pwm_type()) {
```

Comment:
Your definition of is_normal_pwm_type() is not the same as the code you are replacing

Reply:
I think the new definition is correct, this case was just overlooked when the range type was added.